src/node_os.cc

```diff
@@ -149,10 +149,15 @@ static void GetTotalMemory(const FunctionCallbackInfo<Value>& args) {
 
 
 static void GetUptime(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
   double uptime;
   int err = uv_uptime(&uptime);
-  if (err == 0)
-    args.GetReturnValue().Set(uptime);
+  if (err != 0) {
+    env->CollectUVExceptionInfo(args[args.Length() - 1], err, "uv_uptime");
```

Comment:
Is the expectation that users set this argument when calling `.uptime()`?

Reply:
Whoops, nope. I was just following the pattern used in e.g: [`GetHostname`](https://github.com/nodejs/node/blob/main/src/node_os.cc#L61), I guess by adding this diff:

```diff
diff --git a/lib/os.js b/lib/os.js
index 81bbe15a9d..b3a96e65b1 100644
--- a/lib/os.js
+++ b/lib/os.js
@@ -55,7 +55,7 @@ const {
   getOSInformation: _getOSInformation,
   getTotalMem,
   getUserInfo,
-  getUptime,
+  getUptime: _getUptime,
   isBigEndian,
   setPriority: _setPriority
 } = internalBinding('os');
@@ -80,6 +80,8 @@ const {
 const getHomeDirectory = getCheckedFunction(_getHomeDirectory);
 const getHostname = getCheckedFunction(_getHostname);
 const getInterfaceAddresses = getCheckedFunction(_getInterfaceAddresses);
+const getUptime = getCheckedFunction(_getUptime)
+
 /**
  * @returns {string}
  */
```

That would make it work as _really_ expected, right?

Reply:
Yes, I think so. Although I will admit that this `getCheckedFunction()` thing seems a bit bizarre when just throwing errors from C++ is also an option, but obviously that's not something for this PR.

Reply:
Thanks for the review <3

I felt the same thing. I don't fully understand how that function works... I believe the exceptions are being thrown in the JS land, like, when C++ returned "undefined"